view/frontend/requirejs-config.js

```diff
@@ -1,5 +1,7 @@
 var config = {
-    "paths": {
-        "InstagramWidget": "Bitbull_InstagramWidget/js/InstagramWidget"
+    map: {
+        '*': {
+            InstagramWidget: 'Bitbull_InstagramWidget/js/InstagramWidget'
```

Comment:
I'm unsure. For the moment I leave the way you did, then I'll open a thread in the forum